.travis.yml

```diff
@@ -66,6 +66,19 @@ matrix:
         - DESC="package no error"
         - CMD="mvn clean package -Passembly"
 
+    # checkstyle regression no exception (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle regression no exception"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cd checkstyle &&"
+        - CMD3="git checkout -b test-branch &&"
+        - CMD4="git checkout master &&"
+        - CMD5="git reset --hard HEAD^ &&"
```

Comment:
So we will have two pairs: HEAD and HEAD^; one specific commit and the previous one. Is it correct?